### DISCUSSION TITLE
Programatically Import Calendar

### DISCUSSION THREAD

**Discussion by aMediocreDad** [vigoren/foundryvtt-simple-calendar#180] [Ideas]
Hey!

Forbidden Lands used to configure Calendar/Weather programatically on creating a new world in the system. Since About Time has abandoned calendar support and Calendar/Weather relied on it for this, the feature in Forbidden Lands is broken. 

Users are requesting that this feature be fixed, but looking into this I cannot see any good solution, or any entry point using this module (which Calendar/Weather now seems to rely upon for calendar features).

Are you able to provide any assistance in solving this issue?

Cheers!

**Comment by vigoren** [comment]
Sorry didn't respond to this as we spoke on discord!

Yes I can add this and the latest release had a programmatic way to set the calendars configuration now.

The only thing it doesn't do is allow the programmatic saving of notes. Which is something I will add in.

With the addition of forbidden lands as a predefined calendar I think configuring using that may be the easiest for you.

Thanks for the suggestion 😁
